R/create_credentials.R

```diff
@@ -155,12 +155,24 @@ create_smtp_creds_key <- function(id,
   # nocov end
 }
 
+#' Ask for a password
+#'
+#' @noRd
+get_password <- function(msg = "Enter the SMTP server password: ") {
+
+  # nocov start
+
+  getPass::getPass(msg = msg)
+
+  # nocov end
+}
+
 #' Create a credentials list object
 #'
 #' @noRd
 create_credentials_list <- function(provider,
```

Comment:
Just want to confirm, you’ve tried this and it works? Even without get_pass being exported?

Reply:
I did test it with `create_smtp_creds_file()` and it created the JSON file.

Reply:
Oh sorry I missed that this function is itself unexported.